app/build.gradle.kts

```diff
@@ -54,6 +54,8 @@ android {
 }
 
 dependencies {
+    implementation(projects.feature.home)
```

Comment:
Please delete it. Our app module should know only about host screens and splash screen at the moment.